datalake/src/test/java/com/sequenceiq/datalake/flow/start/handler/RdsStartHandlerTest.java

```diff
@@ -84,7 +84,7 @@ public void shouldNotCallStartInCaseNoExternalDatabaseButSetStartedStatus() {
         victim.accept(event);
 
         verify(eventBus).notify(eq(RdsStartSuccessEvent.class.getSimpleName()), any(Event.class));
-        verifyZeroInteractions(databaseService);
+        verifyNoMoreInteractions(databaseService);
```

Comment:
Can we use the `verifyNoInteractions` instead of `verifyNoMoreInteractions` please?

Reply:
Fixed! Changed all `verifyZeroInteractions` -> `verifyNoMoreInteractions` changes to `verifyZeroInteractions` -> `verifyNoInteractions`